src/features/classification/security-controls/SecurityControlsModal.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import type { MessageDescriptor } from 'react-intl';
+
+import { Modal, ModalActions } from '../../../components/modal';
+import Button from '../../../components/button';
+import ClassifiedBadge from '../ClassifiedBadge';
+import Label from '../../../components/label/Label';
+import SecurityControlsItem from './SecurityControlsItem';
+import { CLASSIFICATION_COMMUNITY_URL } from '../constants';
+import { Link } from '../../../components/link';
+import messages from './messages';
+import classificationMessages from '../messages';
+import './SecurityControlsModal.scss';
+
+type Props = {
+    classificationName: string,
+    closeModal: Function,
+    definition: string,
+    isSecurityControlsModalOpen: boolean,
+    itemName: string,
+    modalItems: Array<MessageDescriptor>,
+};
+
+const SecurityControlsModal = ({
+    closeModal,
+    definition,
+    classificationName,
+    isSecurityControlsModalOpen,
+    itemName,
+    modalItems,
+}: Props) => {
+    const title = <FormattedMessage {...messages.modalTitle} values={{ itemName }} />;
+
+    const learnMoreLink = (
+        <Link
+            data-resin-target="communityUrl"
+            href={CLASSIFICATION_COMMUNITY_URL}
+            rel="noopener noreferrer"
+            target="_blank"
+        >
+            <FormattedMessage {...messages.learnMoreLink} />
+        </Link>
+    );
+
+    return (
+        <Modal
+            className="bdl-SecurityControlsModal"
+            title={title}
+            onRequestClose={closeModal}
+            isOpen={isSecurityControlsModalOpen}
+        >
+            <p>
+                <FormattedMessage {...messages.modalDescription} values={{ learnMoreLink }} />
+            </p>
+            <ClassifiedBadge name={((classificationName: any): string)} />
+            <Label text={<FormattedMessage {...classificationMessages.definition} />}>
+                <p className="bdl-Classification-definition">{definition}</p>
+            </Label>
+            <ul className="bdl-SecurityControls">
```

Comment:
Thanks @jstoffan , I left this as-is bc securitycontrolsmodal is utilizing securityControlsItem (not securityControls) so it would not create a circular dependency. As discussed with @patrickpaul, we were thinking to create two separate child components for `list of securityControlsItem` and `viewAll button + modal` if that would make it cleaner. But as implementing, I realized it would create additional layers/props checking,  yet the logic in security controls component is not made simpler. Although that `list of securityControlsItem` is also used in the modal, we have assigned different classnames on the ul element. Thus I think it makes sense to keep them separated.